nextflow.config

```diff
@@ -9,35 +9,84 @@
 params {
 
   // Workflow flags
-  // TODO nf-core: Specify your pipeline's command line flags
-  reads = "data/*{1,2}.fastq.gz"
-  singleEnd = false
+  annotateTools = null // Only with --step annotate
+  genome = 'GRCh38'
+  input = null // No default input
+  noGVCF = null // g.vcf are produced by HaplotypeCaller
+  noStrelkaBP = null // Strelka will use Manta candidateSmallIndels if available
+  skipQC = null // All QC tools are used
+  step = 'mapping' // Starts with mapping
+  tools = null // No default Variant Calling or Annotation tools
+
+  // Workflow settings
+  annotation_cache = null // Annotation cache disabled
+  cadd_cache = null // CADD cache disabled
+  genesplicer = null // genesplicer disabled
+  markdup_java_options = '"-Xms4000m -Xmx7g"' //Established values for markDuplicate memory consumption, see issue PR #689 for details
```

Comment:
Hah! I managed to find an error. This issue nr refers to the scilifelab/sarek repo. Should probably switch this to a link to that specific issue instead.

Reply:
This is the source issue, so it'll always be this one that is referred to ;-)

Reply:
You mean this is from the template as well? I guess when nf-core/sarek reaches #689, this will be confusing.

Reply:
No, I meant the docs part.
But your right, I should add the right url for this issue

Reply:
Yes, just adding the repo org/name prefix to the issue number should be sufficient 👍🏻